src/test/resources/FirstImportComment.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Some comment
+ *
+ */
+import java.lang.Math;
```

Comment:
This isn't a good test, because this is a javadoc comment, and import statements cannot have javadoc comments. So, this would be a dangling javadoc comment, not associated with any actual line of code, and it would be expected to insert a blank line between the dangling comment and the start of the import statements.

I'm curious if the issue still applies if it is a multi-line non-javadoc comment or an inline comment above the line.

Reply:
Hey @ctubbsii, seems like this only occurs with a multiline non-javadoc comment as a single inline comment above is handled correctly. I've updated the test case to reflect a valid use-case.